manifests/agent/config.pp

```diff
@@ -48,24 +48,4 @@
   $puppet::agent_additional_settings.each |$key,$value| {
     puppet::config::agent { $key: value => $value }
   }
-
-  if $puppet::runmode == 'service' {
-    $should_start = 'yes'
-  } else {
-    $should_start = 'no'
-  }
-
-  if $facts['os']['family'] == 'Debian' {
-    augeas {'puppet::set_start':
-      context => '/files/etc/default/puppet',
-      changes => "set START ${should_start}",
-      incl    => '/etc/default/puppet',
-      lens    => 'Shellvars.lns',
-    }
-    if $puppet::remove_lock {
```

Comment:
`$puppet::remove_lock` is no longer used anywhere with this change, so the param should be removed entirely from init.pp and params.pp

Reply:
Good point.